000-debug/0-load.php

```diff
@@ -4,3 +4,9 @@
 
 require_once( __DIR__ . '/logger.php' );
 require_once( __DIR__ . '/debug-mode.php' );
+
+function remove_background_updates_test( $tests ) {
+ unset( $tests['async']['background_updates'] );
+ return $tests;
```

Comment:
Tabs over spaces :)

```suggestion
	unset( $tests['async']['background_updates'] );
	return $tests;
```

Reply:
Argh, this always gives me issues on this repo. I turned off auto-indent and manually tabbed myself, but it still gives me issues! My PHPCS never works either -.-. I just committed your suggestion and it seemed to work!